src/main/java/br/com/sbs/cubatech/category/api/CategoryApiController.java

```diff
@@ -17,8 +22,16 @@ public CategoryApiController(CategoryRepository categoryRepository){
     }
 
     @GetMapping(value = "/api/categories", produces ={ MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE } )
+    @Cacheable(value = "categories")
     public ResponseEntity<List<CategoryApiDto>> findActiveCategories(){
         List<Category> categoryList = categoryRepository.findByStatus(Status.ACTIVE);
         return ResponseEntity.ok().body(CategoryApiDto.convertAll(categoryList));
     }
+
+    @GetMapping(value = "/bGltcGEtby1jYWNoZS1kYS1hcGktYWU")
+    @CacheEvict(value = "categories", allEntries = true)
+    public ResponseEntity<?> clearCache(){
```

Comment:
Quando não tem um tipo de retorno, você pode usar `Void`.

```suggestion
    public ResponseEntity<Void> clearCache(){
```

Reply:
Ajustado